test/ios/mocked_data/test_get_mac_address_table/isr_multiple_ehwic_format/expected_result.json

```diff
@@ -0,0 +1,18 @@
+[{
+	"static": false,
+	"active": true,
+	"vlan": 2,
+	"mac": "11:11:22:22:33:33",
+	"interface": "GigabitEthernet0/0/1",
+	"moves": -1,
+	"last_move": -1.0
+},
+{
+	"static": false,
+	"active": true,
+	"vlan": 1,
+	"mac": "3333.2222.1111",
```

Comment:
I had the feeling I've already commented on this previously - looks like I didn't... Yes, you placed the mock data correctly, just a minor correction: https://travis-ci.org/github/napalm-automation/napalm/jobs/661437097#L513 so the MAC should be formatted as `33:33:22:22:11:11` on this line.